src/examples/Text.cpp

```diff
@@ -0,0 +1,273 @@
+    shape->fill(75, 75, 75);
+    canvas->push(std::move(shape));
+
+    //Load a necessary font data.
+    //The loaded font will be released when the Initializer::term() is called.
+    //Otherwise, you can immedately unload the font data.
+    //Please check Text::unload() APIs.
+    if (tvg::Text::load(EXAMPLE_DIR"/Arial.ttf") != tvg::Result::Success) {
+        cout << "TTF is not supported. Did you enable TTF Loader?" << endl;
+        return;
+    }
+
+    if (tvg::Text::load(EXAMPLE_DIR"/NanumGothicCoding.ttf") != tvg::Result::Success) {
+        cout << "TTF is not supported. Did you enable TTF Loader?" << endl;
+        return;
+    }
+
+    if (tvg::Text::load(EXAMPLE_DIR"/SentyCloud.ttf") != tvg::Result::Success) {
+        cout << "TTF is not supported. Did you enable TTF Loader?" << endl;
+        return;
+    }
+
+    auto text = tvg::Text::gen();
+    text->font("Arial", 80);
+    text->text("THORVG Text");
+    text->fill(255, 255, 255);
+    canvas->push(std::move(text));
+
+    auto text2 = tvg::Text::gen();
+    text2->font("Arial", 30, "italic");
+    text2->text("Font = \"Arial\", Size = 40, Style=Italic");
+    text2->translate(0, 150);
+    text2->fill(255, 255, 255);
+    canvas->push(std::move(text2));
+
+    auto text3 = tvg::Text::gen();
+    text3->font("Arial", 40);
+    text3->text("Kerning Test: VA, AV, TJ, JT");
+    text3->fill(255, 255, 255);
+    text3->translate(0, 225);
+    canvas->push(std::move(text3));
+
+    auto text4 = tvg::Text::gen();
+    text4->font("Arial", 25);
+    text4->text("Purple Text");
+    text4->fill(255, 0, 255);
+    text4->translate(0, 310);
+    canvas->push(std::move(text4));
+
+    auto text5 = tvg::Text::gen();
+    text5->font("Arial", 25);
+    text5->text("Gray Text");
+    text5->fill(150, 150, 150);
+    text5->translate(220, 310);
+    canvas->push(std::move(text5));
+
+    auto text6 = tvg::Text::gen();
+    text6->font("Arial", 25);
+    text6->text("Yellow Text");
+    text6->fill(255, 255, 0);
+    text6->translate(400, 310);
+    canvas->push(std::move(text6));
+
+    auto text7 = tvg::Text::gen();
+    text7->font("Arial", 15);
+    text7->text("Transformed Text - 30'");
+    text7->fill(0, 0, 0);
+    text7->translate(600, 400);
+    text7->rotate(30);
+    canvas->push(std::move(text7));
+
+    auto text8 = tvg::Text::gen();
+    text8->font("Arial", 15);
+    text8->fill(0, 0, 0);
+    text8->text("Transformed Text - 90'");
+    text8->translate(600, 400);
+    text8->rotate(90);
+    canvas->push(std::move(text8));
+
+    auto text9 = tvg::Text::gen();
+    text9->font("Arial", 15);
+    text9->fill(0, 0, 0);
+    text9->text("Transformed Text - 180'");
+    text9->translate(800, 400);
+    text9->rotate(180);
+    canvas->push(std::move(text9));
+
+    //gradient texts
+    float x, y, w, h;
+
+    auto text10 = tvg::Text::gen();
+    text10->font("Arial", 50);
+    text10->text("Linear Text");
+    text10->translate(0, 350);
+    text10->bounds(&x, &y, &w, &h);
```

Comment:
Please see the api: 
![image](https://github.com/thorvg/thorvg/assets/3711518/efae57a9-224e-406c-8cc5-26a0b4d768e9)

Reply:
um I checked that x, y, w, and h looks normally when transformed is true.
However, I think.. it seem a little strange.

case 1
```cpp
    auto text10 = tvg::Text::gen();
    text10->font("Arial", some_size);
    text10->text("Some text");
    text10->bounds(&x, &y, &w, &h, false);
```

case 2
```cpp
    auto text10 = tvg::Text::gen();
    text10->font("Arial", some_size);
    text10->text("Some text");
    text10->bounds(&x, &y, &w, &h, true);
```

In both cases, the user did not use a transform API such as translate, but the resulting values of bounds() are different.
(assuming there is no parent)
The shapes of the text will perform a transformation internally, but it will not be a user-defined value.
Shape inputs its size directly, so the use of bound's transformed is intuitive.
However, since text is determined internally, I think the bounds returned to the user should have a separate original bounds.
Gradient is applied based on the internally transformed size of the text. Therefore, the expected results may different.
Of course, we can inform about these precautions in docs of text.fill().